src/Miunie.Discord/Logging/DiscordLogger.cs

```diff
@@ -13,16 +13,18 @@ public DiscordLogger(ILogWriter logger)
             _logger = logger;
         }
 
-        internal void Log(object sender, DebugLogMessageEventArgs e)
+        internal Task Log(LogMessage evt)
```

Comment:
> `(LogMessage evt)`

What does evt stand for? - isn't `(LogMessage log)` better in this context?

Reply:
Either that or args.